### ISSUE TITLE
Just a query

### ISSUE THREAD

**Issue by harish0201** [jokergoo/circlize#133]
Hi,

I'm using the package for the first time, so I have a doubt. I'm attaching a file that is my dataframe on which I'm trying to plot.

[The file.](https://github.com/jokergoo/circlize/files/3612705/plot.txt)

Can you help me understand what triggers this error:
`Error in rep(small.gap, n_df1 - 1) : invalid 'times' argument`

Thank you!

**Comment by jokergoo** [comment]
Can you also attach the code you use?

**Comment by lixiaopi1985** [comment]
I am having the same error with one of the dataset. I have compared it to another dataset which is working fine. I could not debug where went wrong but they look like pretty much the same structurally. I extracted the grouping and ran it as lapply(tg, function(x)c(rep(1, x-1), 2)), no errors.

the function and arguments I used:

chordDiagram(df, 
             annotationTrack = "grid", 
             grid.col = sector_colors, 
             col = linkcolors,
             preAllocateTracks = list(track.height = uh(5, "mm"),
                                      track.margin = c(uh(2, "mm"), 0)),
             group = group,
             scale = T,
             big.gap = 2)


my files: 

[notworking](https://drive.google.com/file/d/1bKd-zpqYAg8qsV3PaF_2WmUoPnWwQd6N/view?usp=sharing)

[working](https://drive.google.com/file/d/1bKd-zpqYAg8qsV3PaF_2WmUoPnWwQd6N/view?usp=sharing)

those are rds in a list format with df, sector colors, link colors and groups stored.

Thank you!
